pkgs/development/libraries/libsmu/libsmu-pc.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/dist/libsmu.pc.cmakein b/dist/libsmu.pc.cmakein
+index f89dad9..4ad9b0a 100644
+--- a/dist/libsmu.pc.cmakein
++++ b/dist/libsmu.pc.cmakein
+@@ -1,7 +1,7 @@
+ prefix=@CMAKE_INSTALL_PREFIX@
+ exec_prefix=${prefix}
+-libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
+-includedir=${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
++libdir=@CMAKE_INSTALL_FULL_LIBDIR@
++includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
```

Comment:
Would be great to upstream this

Reply:
jup
https://github.com/analogdevicesinc/libsmu/pull/206